src/features/SensorStatus/SensorStatusManager.ts

```diff
@@ -98,6 +103,19 @@ export class SensorStatusManager {
     };
   };
 
+  getAllStatuses = async (): Promise<ById<SensorStatus>> => {
+    const query = buildSensorStatusQuery('isActive = 1');
+    const result = (await this.databaseService.query(query)) as SensorStatusQueryResult[];
+    const mappedResults = result.map(queryResult => this.mapStatusQueryResult(queryResult));
+
+    const indexed: ById<SensorStatus> = {};
+    mappedResults.forEach(sensorStatus => {
+      indexed[sensorStatus.id] = sensorStatus;
+    });
+
+    return indexed;
+  };
```

Comment:
added this querying which is fetching all the sensor statuses at the same time for initializing